### PR TITLE
Make iframe dropdown overlay

### LINKED ISSUE #30: per iframe dropdowns should be overlayed
The per iframe dropdowns should be overlaid, not discrete blocks/inserts in the sidebar.

2Capture_27

### PR DESCRIPTION
**Pull request by chrisplatorres**
This PR makes the iframe dropdown overlay.

Closes #30

### CODE REVIEW

File: ui/src/css/dom.css [hunk 1]
```diff
@@ -89,6 +89,8 @@
   padding-left: 10px;
   display: table-cell;
   vertical-align: middle;
+  position: absolute;
+  z-index: 1000;
```
Comment (modulesio) [inline ui/src/css/dom.css]: Can it be `1`?

Comment (modulesio) [reply ui/src/css/dom.css]: Or maybe even `0`?

Comment (chrisplatorres) [reply ui/src/css/dom.css]: Yes `1` works, but `0` doesn't. e46f9b1

Comment (chrisplatorres) [reply ui/src/css/dom.css]: ( heh, what a *binary* answer 🥁 )

**Pull request merged**